packages/amplify_authenticator/lib/src/l10n/input_resolver.dart

```diff
@@ -423,30 +433,37 @@ class InputResolver extends Resolver<InputResolverKey> {
     }
   }
 
+  /// Returns the [String] displayed as the input hint.
   String hint(BuildContext context, InputField field) {
     final fieldName = title(context, field);
     final lowercasedFieldName = fieldName.toLowerCase();
     return AuthenticatorLocalizations.inputsOf(context)
         .promptFill(lowercasedFieldName);
   }
 
+  /// Returns the [String] used as the hint for the password confirmation field.
   String confirmHint(BuildContext context, InputField field) {
     final fieldName = AuthenticatorLocalizations.inputsOf(context).password;
     final lowercasedFieldName = fieldName.toLowerCase();
     return AuthenticatorLocalizations.inputsOf(context)
         .promptRefill(lowercasedFieldName);
   }
 
+  /// Returns the [String] displayed when a required field is left empty.
   String empty(BuildContext context, InputField field) {
     return AuthenticatorLocalizations.inputsOf(context)
         .warnEmpty(title(context, field));
   }
 
+  /// Returns the [String] displayed when a field fails a format validation check,
+  /// such as an invalid email format, an invalid confirmation code length, etc.
   String format(BuildContext context, InputField field) {
     return AuthenticatorLocalizations.inputsOf(context)
         .warnInvalidFormat(title(context, field).toLowerCase());
   }
 
+  /// Returns the [String] displayed when a password input does match the password requirements
+  /// defined in the amplifyconfiguration.dart file.
```

Comment:
```suggestion
  /// defined in the configuration.
```

Not necessarily in the file if they are building configs by hand.